tests/skip.json

```diff
@@ -19,6 +19,7 @@
     "built-ins/Array/S15.4_A1.1_T10.js",
     "built-ins/Array/S15.4.5.2_A1_T1.js",
     "built-ins/Array/S15.4.5.2_A3_T3.js",
+    "built-ins/String/prototype/repeat/repeat-string-n-times.js",
```

Comment:
**question:** Why is this skipped? If it's simply because the engine's performance is not good enough to do the work within 10 seconds then I'd say leave it unskipped. It's then a valid reminder to us that this is too slow :)

Reply:
Yeah, it's because the while loop or comparison in the test takes too long. The operation itself is fast though.

Reply:
I'll create a perf issue about that. Please remove the skip, let's let it annoy us so that we're forced to fix it :)